CodeEdit/InspectorSidebar/Extensions/String+MD5.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  String+MD5.swift
+//  
+//
+//  Created by Nanashi Li on 2022/04/19.
+//
+
+import Foundation
+import CryptoKit
+
+extension String {
+    var MD5: String {
```

Comment:
Add documentation & properties should start with lowercase letter according to Swift style guidelines.

```swift
var md5: String {}
```